src/server/sql/group/create_groups_tables.sql

```diff
@@ -154,8 +154,24 @@ CREATE FUNCTION check_cyclic_groups()
 	$$
 	LANGUAGE 'plpgsql';
 
-CREATE TRIGGER check_cyclic_groups_trigger
+-- This will avoid trying to create the trigger if it exists as that causes an error. This is an issue since
+-- the OED install stops the creation of database items after this.
+DO
+$$
+BEGIN
+IF NOT EXISTS(SELECT *
```

Comment:
I'm unclear on what problem this might cause. Can you elaborate on what the concern is? Also, while exception handling can be used in some circumstances, it can be heavier weight and, when possible, it seems better to deal with directly rather than as an error.